cluster/examples/kubernetes/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -33,22 +33,6 @@ spec:
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
-        - name: csi-rbdplugin-attacher
```

Comment:
Should this be left but commented out with a note that it is required for k8s 1.13.x and below?

Reply:
same explanation as https://github.com/rook/rook/pull/4172#discussion_r342387683